src/utils/probe-utils.js

```diff
@@ -149,7 +149,7 @@ export function zipByAggregationType(payload) {
 export function topKBuildsPerDay(dataset, k = 2) {
   const byBuildID = groupBy(dataset, 'label', formatBuildIDToOnlyDate);
   const topK = Object.entries(byBuildID).map(([_, matches]) => {
-    const out = matches.filter((m) => m.audienceSize > 10);
+    const out = matches;
```

Comment:
here is an example of a downstream bug. The filtering here actually created an issue where data was missing because we were filtering out (for some reason, likely historical ones) data points with a small audience size.